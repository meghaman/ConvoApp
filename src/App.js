import React from 'react';

import Graph from '@Components/graph'
import AddPersonButton from '@Components/buttons/addPersonButton'
import StartConversationButton from '@Components/buttons/startConversationButton'
import StopConversationButton from '@Components/buttons/stopConversationButton'
 
const App = () =>
{
  return(
    <div>
        <Graph></Graph>
        <br />
        <AddPersonButton></AddPersonButton>
        <StartConversationButton></StartConversationButton>
        <StopConversationButton></StopConversationButton>
    </div>
  )
};

export default App;