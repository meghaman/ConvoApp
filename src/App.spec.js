import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';
 
describe('Basic App Snapshot', () => {
  test('snapshot renders', () => {
    const title = 'Convo App';

    const component = renderer.create(<App title={ title } />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});