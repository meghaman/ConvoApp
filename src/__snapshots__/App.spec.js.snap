// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic App Snapshot snapshot renders 1`] = `
<div>
  <svg
    height={500}
    width={500}
  >
    <g
      transform="translate(250 250)"
    />
  </svg>
  <br />
  <button
    onClick={[Function]}
  >
    Add Person
  </button>
  <button
    onClick={[Function]}
  >
    Start Conversation
  </button>
  <button
    onClick={[Function]}
  >
    Stop Conversation
  </button>
  <div />
</div>
`;
